import.go

```diff
@@ -105,6 +106,15 @@ func WithImportCompression() ImportOpt {
 	}
 }
 
+// WithDiscardUnpackedLayers allows the garbage collector to clean up
+// layers from content store after unpacking.
+func WithDiscardUnpackedLayers(discard bool) ImportOpt {
```

Comment:
I'd say the API should follow the lead of WithImportCompression, where just supplying this option sets it to true and get rid of the bool.

Reply:
I must have been following the implementation of the WithAllPlatforms function. I think you're right, it makes more sense here to remove the bool parameter. Thank you.